test/knight-test.js

```diff
@@ -0,0 +1,169 @@
+import { Board } from  "../js/board.js";
+import { Knight } from "../js/knight.js";
+
+/**
+ * Test suite for the Knight class.
+ */
+QUnit.module("Knight", hooks => {
+    let board;
+
+    hooks.beforeEach(() => {
+        board = new Board();
+    });
+
+    hooks.afterEach(() => {
+        board.reset();
+    });
+
+    /**
+     * Test Constructor
+     */
+    QUnit.test("Constructor", assert => {
+        let knight = new Knight(6969, 0, 0, 0);
+
+        assert.deepEqual(
+            knight.getId(),
+            6969,
+            "Verify {id} is 6969."
+        );
+        assert.deepEqual(
+            knight.getRowPos(),
+            0,
+            "Verify {row} is 0."
+        );
+        assert.deepEqual(
+            knight.getColPos(),
+            0,
+            "Verify {column} is 0."
+        );
+        assert.deepEqual(
+            knight.playerId,
+            0,
+            "Verify {playerId} is 0."
+        );
+        assert.deepEqual(
+            knight.unicodeChar,
+            "U+265E",
```

Comment:
Unicode character on line `46` should be `&#9822;`.

Reply:
Updated.